daemon/execdriver/driver.go

```diff
@@ -24,10 +24,16 @@ var (
 	ErrDriverNotFound          = errors.New("The requested docker init has not been found")
 )
 
-// StartCallback defines a callback function.
+const (
```

Comment:
also, there is no such a thing as `PostStart` event, it's `PostStop` :stuck_out_tongue_winking_eye: